apps/api/src/app/models/users/users.service.ts

```diff
@@ -72,11 +76,12 @@ export class UsersService {
   }
 
   async updateSettings(settingsDto: SettingsDto): Promise<User> {
-    const userFromDb = await this.userModel.findOne({ email: settingsDto.email});
+    const userFromDb = await this.model.findOne({ email: settingsDto.email});
     if(!userFromDb) throw new HttpException('COMMON.USER_NOT_FOUND', HttpStatus.NOT_FOUND);
     
     userFromDb.settings = userFromDb.settings || {};
     for (const key in settingsDto) {
+      // eslint-disable-next-line no-prototype-builtins
```

Comment:
What means this comment?

Reply:
> What means this comment?

don't use it :D on this case type script don't understand hasOwnProperty, and this line disabling eslint error for current line